### PR TITLE
First open-source contribution.

### PR DESCRIPTION
**Pull request by George-prog275**
Added a new card.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
           <!-- Keep one line of space above and below your card -->
           <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+          <!-- ________ *TEMPLATE: MAKE A COPY* George's card START ________  -->
+          <div class="card">
+            <p class="name">George Baraza</p>
+            <p class="contact">
+                <i class="fab fa-twitter"></i>
+                <a href="https://www.twitter.com/" target="_blank">@Georginio_b</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
                <a href="https://www.twitter.com/Georginio_b" target="_blank">@Georginio_b</a>
```

File: index.html [hunk 2]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
...
+                <p>3 Useful Dev Resources</p>
+                <ul>
+                    <li>
+                        <a href="#" target="_blank" title="First Resource">Mozilla Developer Network</a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank" title="Second Resource">W3Schools</a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank" title="Third resource">JavaScript Algorithms & Data Structures by Michael McMillan</a>
```
Comment (TenTraicion) [inline index.html:136]: Issue: No Links

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@George-prog275, Your card is lacking proper links. kindly fix your social/contact link, it should lead to your twitter profile. and also kindly add the resource links which should lead to the desired website. Thanks!

**Comment by George-prog275** [comment]
Just rectified the mistakes I made.

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@George-prog275, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**